.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,139 @@
+name: Benchmark Tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches: ['master']
+
+permissions:
+  pull-requests: write
```

Comment:
Looks like the check failure is expected for now, and should be fixed once the PR is merged since the permissions look good :+1: 

![image](https://user-images.githubusercontent.com/591645/132630858-ab944557-2c31-4ec7-b3d2-a24935d0dae7.png)

Reply:
(as mentioned in https://github.com/jupyterlab/jupyterlab/pull/10936#issuecomment-908433652)